composer.json

```diff
@@ -12,40 +12,39 @@
     "require" : {
         "php": "^5.6 || ^7.0",
         "cocur/slugify": "^1.4 || ^2.5",
-        "friendsofsymfony/comment-bundle": "^2.0.13",
         "friendsofsymfony/rest-bundle": "^1.8 || ^2.2",
-        "friendsofsymfony/user-bundle": "^1.3",
+        "friendsofsymfony/user-bundle": "^1.3 || ^2.0",
         "jms/serializer-bundle": "^0.13 || ^1.5",
         "knplabs/knp-menu-bundle": "^1.1 || ^2.1",
         "knplabs/knp-paginator-bundle": "^2.6",
         "kriswallsmith/buzz": "^0.15",
         "nelmio/api-doc-bundle": "^2.13",
-        "sonata-project/admin-bundle": "^3.23",
+        "sonata-project/admin-bundle": "^3.28",
         "sonata-project/block-bundle": "^3.4",
         "sonata-project/classification-bundle": "^3.3",
-        "sonata-project/comment-bundle": "^3.0",
-        "sonata-project/core-bundle": "^3.5",
-        "sonata-project/datagrid-bundle": "^2.2.1",
-        "sonata-project/doctrine-orm-admin-bundle": "^3.1.7",
+        "sonata-project/comment-bundle": "^3.1",
+        "sonata-project/core-bundle": "^3.7.1",
+        "sonata-project/datagrid-bundle": "^2.3",
+        "sonata-project/doctrine-orm-admin-bundle": "^3.2",
         "sonata-project/formatter-bundle": "^3.2",
-        "sonata-project/intl-bundle": "^2.3",
-        "sonata-project/media-bundle": "^3.7",
+        "sonata-project/intl-bundle": "^2.4",
+        "sonata-project/media-bundle": "^3.10",
         "sonata-project/notification-bundle": "^3.2",
-        "sonata-project/seo-bundle": "^2.2",
-        "sonata-project/user-bundle": "^3.5",
-        "symfony/config": "^2.8",
-        "symfony/console": "^2.8",
-        "symfony/form": "^2.8",
-        "symfony/http-foundation": "^2.8",
-        "symfony/process": "^2.8",
-        "symfony/routing": "^2.8",
-        "symfony/security-bundle": "^2.8",
-        "symfony/twig-bridge": "^2.8",
-        "symfony/validator": "^2.8"
+        "sonata-project/seo-bundle": "^2.4",
+        "sonata-project/user-bundle": "^3.5 || ^4.0",
```

Comment:
Probably until user-bundle 4.0 comes out you need to add ^4.0@dev

Reply:
Not at this moment, maybe today will be

Reply:
@jordisala1991 can you help?
I just tried to update composer with `"sonata-project/user-bundle": "^4.0@dev"` and `"sonata-project/user-bundle": "dev-master"` and got this:
```
    - sonata-project/user-bundle dev-master conflicts with sonata-project/intl-bundle[2.4.0].
    - sonata-project/intl-bundle 2.4.0 conflicts with sonata-project/user-bundle[dev-master].
    - don't install sonata-project/user-bundle 4.x-dev|install sonata-project/user-bundle dev-master
    - Installation request for sonata-project/user-bundle 4.x-dev -> satisfiable by sonata-project/user-bundle[4.x-dev].
    - Installation request for sonata-project/intl-bundle ^2.4 -> satisfiable by sonata-project/intl-bundle[2.4.0].
```
I checked: intl-bundle has
```
"conflict": {
        "sonata-project/user-bundle": "<2.0 || >=5.0"
    },
```
Why does composer throw this error?

Reply:
because intl-bundle depends on user-bundle and version 4.0 is not out yet, try ^3.0 || ^4.0@dev as 3.x-dev